Exploration Queries/InputEntity_Account/UserAccount_SubsOwnedViaAadGroups.yaml

```diff
@@ -0,0 +1,35 @@
+Id: 3ffccd66-95b7-4437-9487-0297b40203e8
+DisplayName: "Subscriptions transitively owned by the user via AAD Group memberships"
+Description: "Returns all subscriptions owned transitively by the user via Azure Active Directory Group memberships"
+InputEntityType: Account 
+InputFields:
+  - AadUserId 
+OutputEntityTypes:
+  - AzureResource  
+QueryPeriodBefore: 7d 

```

Comment:
Normally we want only 12h before and 12h after, 7d lookback in an expansion query but cause the graph to hang